### PR TITLE
Switch syntax highlighting to PrismJS

### LINKED ISSUE #3682: JavaScript syntax highlighting breaks
The title should be concise enough. Tested against `1324d62 `.

screen shot 2018-03-16 at 22 48 41

The comment highlighting continues until EOF.

### PR DESCRIPTION
**Pull request by lafriks**
Looks to be working fine except for diff views that still needs to be fixed. Would need some help on testing more source code to see if it does not break.

Fixes #3682

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

File: modules/highlight/highlight.go [hunk 1]
```diff
@@ -26,45 +26,63 @@ var (
...
+		".yml":   "yaml",
+		".ino":   "arduino",
+		".kt":    "kotlin",
+		".ktm":   "kotlin",
+		".kts":   "kotlin",
+		".m":     "matlab",
+		".mlx":   "matlab",
+		".pas":   "pascal",
+		".patch": "diff",
+		".py":    "python",
```
Comment (lunny) [inline modules/highlight/highlight.go:76]: There is no `.go` support?

**Review by lunny — CHANGES_REQUESTED** [review]

File: modules/highlight/highlight.go (hunk 1 above)
Comment (lafriks) [reply modules/highlight/highlight.go:76]: There is, just in other dict

**Review by lunny — APPROVED** [review]

**Comment by lafriks** [comment]
I have fixed diff view but currently changed line part highlight is still lost

**Comment by lafriks** [comment]
That has to be tested

**Comment by techknowlogick** [comment]
Closing due to performance reasons, and also because we will be switching to server side rendering (likely using chroma)

**Comment by GitMensch** [comment]
> Closing [...] because we will be switching to server side rendering (likely using chroma)

Reference: #7729.

**Pull request closed without merging**